powerquery-docs/CommonIssues.md

```diff
@@ -19,10 +19,11 @@ For example, if you sort a sales table so that each store's largest sale is show
 
 Because of the way Power Query optimizes certain operations, including skipping them or offloading them to data sources (which can have their own unique ordering behavior), sort order isn't guaranteed to be preserved through aggregations (such as `Table.Group`), merges (such as `Table.NestedJoin`), or duplicate removal (such as `Table.Distinct`).
 
-There are a number of ways to work around this. Here are two suggestions:
+There are a number of ways to work around this. Here are few suggestions:
 
 * Perform a sort *after* applying the downstream operation. For example, when grouping rows, sort the nested table in each group before applying further steps. Here's some sample M code that demonstrates this approach: `Table.Group(Sales_SalesPerson, {"TerritoryID"}, {{"SortedRows", each Table.Sort(_, {"SalesYTD", Order.Descending})}})`
 * Buffer the data (using `Table.Buffer`) before applying the downstream operation. In some cases, this operation will cause the downstream operation to preserve the buffered sort order.
+* Use [ranking](/rank-column). For example, instead of using `Table.Distinct`, you could order by the column containing the duplicate values, rank based on a tie-breaker column (such as `modified_date`), and then filter to keep just the rank 1 rows.
```

Comment:
"order by the column**(s)**" to catch the case where there are multiple columns to use for the distinct values.